flexget/plugins/clients/deluge.py

```diff
@@ -317,8 +367,12 @@ def on_get_torrents_status(torrents):
             client.disconnect()
 
         filter = config.get('filter', {})
+
+        extra_keys = config.get('keys', [])
+        combined_list = self.settings_map.keys() + extra_keys
```

Comment:
I know deluge doesn't work in py3, but this particular line won't work in py3 either iirc. Need to convert `self.settings_map.keys()` to a list first.
Also, is there even a need for that variable, `extra_keys`? Couldn't it just be one line?